packages/components/src/interfaces/internal/Notification.ts

```diff
@@ -2,26 +2,25 @@ import type { Static } from "@sinclair/typebox"
 import { Type } from "@sinclair/typebox"
 
 import type { IsomerSiteProps, LinkComponentType } from "~/types"
-import { TextSchema } from "../native/Text"
+import { SimpleProseSchema } from "../native/Prose"
 
 export const NotificationSchema = Type.Object(
   {
-    title: Type.Optional(
-      Type.String({
-        title: "Notification title",
-        description: "The title of the notification",
-        format: "hidden",
-      }),
-    ),
-    content: Type.Array(TextSchema, {
-      title: "Notification content",
-      description: "The content of the notification",
-      format: "hidden",
+    enabled: Type.Boolean({
+      title: "Display a banner",
+      description:
+        "The site notification will always be visible on the site until it is dismissed by the user.",
+      default: false,
     }),
+    title: Type.String({
+      title: "Notification title",
+      description: "The title of the notification",
+      maxLength: 100,
```

Comment:
quick sense check re: discussion on character limit: we are setting a character limit on this?

Reply:
no explicit guidance given on design side but i just set first. i'll check w/ @sehyunidaaa but i retained existing guidelines